omnibus/files/private-chef-cookbooks/private-chef/recipes/postgresql-external.rb

```diff
@@ -20,4 +20,5 @@
   include_recipe "private-chef::erchef_database"
   include_recipe "private-chef::bifrost_database"
   include_recipe "private-chef::oc_id_database"
+  include_recipe "private-chef::bookshelf_database"
```

Comment:
Yes.  Currently the bookshelf code starts sqerl and connects to the database regardless of the config, so we always bootstrap the database.